public/gameInfo.php

```diff
@@ -835,18 +818,31 @@ function submitSetRequest(user, gameID) {
                     echo "</div>";
                 }
 
-                if ($user !== null && $numAchievements > 0) {
-                    echo "<a href='/user/$user'>$user</a> has won <b>$numEarnedCasual</b> achievements";
-                    if ($totalEarnedCasual > 0) {
-                        echo ", worth <b>$totalEarnedCasual</b> <span class='TrueRatio'>($totalEarnedTrueRatio)</span> points";
+                if ($numAchievements > 0) {
+                    echo "<b>Authors:</b> ";
+                    $numItems = count($authorInfo);
+                    $i = 0;
+                    foreach ($authorInfo as $author => $achievementCount) {
+                        echo GetUserAndTooltipDiv($author, false);
+                        echo " (" . $achievementCount . ")";
+                        if (++$i === $numItems) {
+                            echo '.';
+                        } else {
+                            echo ', ';
+                        }
                     }
-                    echo ".<br>";
+                    echo "<br>";
+                    echo "<br>";
+                }
+
+                if ($user !== null && $numAchievements > 0) {
                     if ($numEarnedHardcore > 0) {
-                        echo "<a href='/user/$user'>$user</a> has won <b>$numEarnedHardcore</b> HARDCORE achievements";
-                        if ($totalEarnedHardcore > 0) {
-                            echo ", worth a further <b>$totalEarnedHardcore</b> points";
+                        echo "You've earned <b>$numEarnedHardcore</b> HARDCORE achievements, worth <b>$totalEarnedHardcore</b> <span class='TrueRatio'>($totalEarnedTrueRatio)</span> points.<br>";
+                        if ($numEarnedCasual > 0) { // Some Hardcore earns
+                            echo "You've also earned <b> $numEarnedCasual </b> SOFTCORE achievements worth <b>$totalEarnedCasual</b> points.<br>";
```

Comment:
Recommend `You have earned`... Contractions are generally harder for non-native speakers, and make the site feel just a little less professional.